xwiki/content.md

```diff
@@ -54,6 +54,12 @@ docker run --net=xwiki-nw --name mysql-xwiki -v /my/own/mysql:/var/lib/mysql -e
 
 You should adapt the command line to use the passwords that you wish for the MySQL root password and for the xwiki user password.
 
+Note: If you're using MySQL < 5.6.6, you'll need to start MySQL with:
+
+```console
+docker run --net=xwiki-nw --name mysql-xwiki -v /my/own/mysql:/var/lib/mysql -e MYSQL_ROOT_PASSWORD=xwiki -e MYSQL_USER=xwiki -e MYSQL_PASSWORD=xwiki -e MYSQL_DATABASE=xwiki -d mysql:5.7 --character-set-server=utf8 --collation-server=utf8_bin
```

Comment:
Should the example be `mysql:5.5` (the oldest `5.6` tag we have is `5.6.17`) ?

Reply:
No it's fine, thanks. I prefer to have the example use a more recent version (5.7). Thanks for asking!